app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -329,16 +347,25 @@ class SettingsFragment : PreferenceFragmentCompat(), CoroutineScope, AccountObse
     override fun onLoggedOut() {
         updateAuthState()
         updateSignInVisibility()
+        displayAccountErrorIfNecessary()
```

Comment:
Not necessary here; this is called after there's no more account locally, so it can't be in a bad state.